defaults/test_default.yaml

```diff
@@ -186,3 +186,4 @@ scylla_apt_keys:
   - '17723034C56D4B19'  # ScyllaDB Package Signing Key 2018 <security@scylladb.com>
   - '5E08FBD8B5D6EC9C'  # ScyllaDB Package Signing Key 2020 <security@scylladb.com>
   - 'D0A112E067426AB2'  # ScyllaDB Package Signing Key 2022 <security@scylladb.com>
+  - '6B2BFD3660EF3F5B'  # # Old key to make Artifacts JOB to work - need to revert this commit when the JOB will use manager2.6
```

Comment:
I think you added the wrong key. it should be `5E08FBD8B5D6EC9C` which is already in the list

Reply:
Done